### DISCUSSION TITLE
grapesjs custom code plugin doesn't act after build react app.

### DISCUSSION THREAD

**Discussion by alexjoe118** [GrapesJS/grapesjs#4827] [Q&A]
Hi, All.
In these days, I developed react app using grapesjs.
Everything is fine, but while trying to use custom code plugin, it is act normally with <npm start >comman.
But after deploying on the server, only the custom code component is not visible. So I build on my local using <npm run build >command and run this as local server hosting. Then it is not visible just like server side.
Please help this issue.
This is local side screen. It acts well.
![image](https://user-images.githubusercontent.com/109670019/211127032-0c248801-3ba6-4694-9290-0125c6ee9c36.png)

This is server side and after build local side.
![image](https://user-images.githubusercontent.com/109670019/211127061-fc949261-2585-46a4-9227-6966e536d225.png)

**Comment by artf** [comment]
It's hard to say without having any kind of reproducible example. Do you see logs in the console? Are you bundling the plugin with your code?
